### ISSUE TITLE
Add usage considerations to the docs

### ISSUE THREAD

**Issue by lainisourgod** [getomni-ai/zerox#121]
Hi! Very interesting project and nice documentation overall, but I have a lot of questions right from the beginning

- how much does it cost on average? 
- speed
- some comparisons to other methods: when it works better, when worse

I think having this info right in README would help popularise this method 🙌

**Comment by igorlima** [comment]
Indeed, it's an interesting project. Going through your questions, let me try to address them:

- **Cost:** The pricing largely depends on the LLM providers, as it varies across different platforms. This project uses the `litellm` library, simplifying API calls to various LLMs by standardizing inputs and outputs. This library includes [a comprehensive JSON file](https://github.com/BerriAI/litellm/blob/83879d2a3d77e52785586537cd90c87a664bb010/model_prices_and_context_window.json) that outlines the costs for each provider, making it easy to compare. I've also submitted a PR to enhance the README, which includes [a brief cost comparison](https://github.com/getomni-ai/zerox/pull/129) for your convenience.

- **Speed:** In my use cases, when using LLMs via API, the response time is generally consistent and doesn't show significant delays. A one or two-second difference hasn't been an issue for my use cases. If you're considering running a **local LLM**, that's a different scenario, and I recommend checking out [a Medium post that covers this topic](https://towardsdatascience.com/improved-rag-document-processing-with-markdown-426a2e0dd82b). Feel free to start a discussion there!

- **Comparison Among LLMs:** Several tools are dedicated to comparing LLMs, such as [`promptfoo`](https://github.com/promptfoo/promptfoo/pull/380). Additionally, a [Colab Notebook](https://github.com/neuml/txtai/pull/844#issuecomment-2564109555) is available to play with your data.

Your questions are valuable for understanding and learning more about LLMs, and you are more than welcome to continue this discussion.
